src/Evolution/Systems/Cce/Components/CharacteristicEvolution.hpp

```diff
@@ -225,6 +227,7 @@ struct CharacteristicEvolution {
       Actions::RequestNextBoundaryData<
           typename Metavariables::cce_boundary_component,
           CharacteristicEvolution<Metavariables>>,
+      Actions::SendNextTimeToGH<CharacteristicEvolution<Metavariables>>,
```

Comment:
Is it possible to have this action earlier? We want to send the next CCM time to GH as soon as we know it so we do as little waiting as possible. The comment a few lines above suggests that `ChangeStepSize` actually changes the step size, so right after that maybe? Before we request the next data?